CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-- Fix: default express limit to 1Mb instead default 100Kb and allow change it throught a conf env var 'IOTA_EXPRESS_LIMIT' (#827)
+- Fix: allow send multiple measures to CB in a batch (v2/op/update) instead of using multiples single request (iotagent-json#825)
```

Comment:
```suggestion
- Fix: allow send multiple measures to CB in a batch (POTS /v2/op/update) instead of using multiples single request (iotagent-json#825)
```